src/test/java/org/gridsuite/study/server/StudyTest.java

```diff
@@ -1425,6 +1433,23 @@ public void testNetworkModificationSwitch() {
         assertTrue(requests.stream().anyMatch(r -> r.getPath().matches("/v1/networks/" + NETWORK_UUID_STRING + "/switches/switchId\\?group=.*\\&open=true\\&variantId=" + VARIANT_ID_2)));
     }
 
+    @Test
+    public void testGetLoadMapServer() {
+        //create study
+        UUID studyNameUserIdUuid = createStudy("userId", CASE_UUID);
+        UUID rootNodeUuid = getRootNodeUuid(studyNameUserIdUuid);
+
+        //get the load map data info of a network
+        webTestClient.get()
+                .uri("/v1/studies/{studyUuid}/nodes/{nodeUuid}/network-map/loads/{loadId}", studyNameUserIdUuid, rootNodeUuid, LOAD_ID_1)
+                .exchange()
+                .expectStatus().isOk()
+                .expectHeader().contentType(MediaType.APPLICATION_JSON)
+                .expectBody(String.class);
```

Comment:
check body content ?

Reply:
There is no point, I'm already checking that I called the right url and that URL is returning a mocked answer that isn't even touched by the study-server so the result doesn't matter at all as long as it's present.

Reply:
ok